### PR TITLE
chore: update sample apps

### PR DESCRIPTION
**Pull request by 1abhishekpandey**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] Version upgraded (project, README, gradle, podspec etc)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: android-tv/build.gradle [hunk 1]
```diff
@@ -40,7 +40,7 @@ android {
 
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.5.31'
+    implementation 'androidx.core:core-ktx:1.3.0'
```
Comment (itsdebs) [inline android-tv/build.gradle:43]: Why are we downgrading?

File: build.gradle [hunk 2]
```diff
@@ -5,9 +5,9 @@ buildscript {
         maven { url "https://appboy.github.io/appboy-android-sdk/sdk" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31'
-        classpath 'com.google.gms:google-services:4.3.3'
+        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.21'
```
Comment (itsdebs) [inline build.gradle:9]: Use at least 1.7.10

File: dummy-impl/build.gradle [hunk 3]
```diff
@@ -8,13 +8,10 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 29
```
Comment (itsdebs) [inline dummy-impl/build.gradle:10]: check with 33

File: android-tv/build.gradle (hunk 1 above)
Comment (1abhishekpandey) [reply android-tv/build.gradle:43]: There was some build issue, so I used the sample-kotlin app version over here.

Comment (1abhishekpandey) [reply android-tv/build.gradle]: Updated the version to 1.6.0

File: sample-kotlin-integration/build.gradle [hunk 4]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.6.21'
```
Comment (itsdebs) [inline sample-kotlin-integration/build.gradle:2]: should be 1.7.10

File: sample-kotlin/build.gradle [hunk 5]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.6.21'
```
Comment (itsdebs) [inline sample-kotlin/build.gradle:2]: should be 1.7.10 atleast or 1.8.10

File: android-tv/build.gradle [hunk 6]
```diff
@@ -40,7 +40,7 @@ android {
 
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.5.31'
+    implementation 'androidx.core:core-ktx:1.6.0'
```
Comment (itsdebs) [inline android-tv/build.gradle:43]: Please update this too

File: android-tv/build.gradle [hunk 7]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.20"
```
Comment (itsdebs) [inline android-tv/build.gradle:8]: Please update this

**Review by itsdebs — APPROVED** [review]

**Pull request merged**